src/main/java/io/r2dbc/mssql/MssqlConnection.java

```diff
@@ -197,7 +204,7 @@ Client getClient() {
         return this.client;
     }
 
-    private static String getIsolationLevelSql(IsolationLevel isolationLevel) {
+    private static String getIsolationLevelSql(MssqlIsolationLevel isolationLevel) {
```

Comment:
We could probably get rid of this method and use `MssqlIsolationLevel.asSql()`. What do you think?

Reply:
totally agree.